### ISSUE TITLE
Pix fora do ar?

### ISSUE THREAD

**Issue by grudnitzki** [bacen/pix-perguntas-e-respostas#475]
O Pix caiu ou foi só para mim?

Não consigo nem fazer transferência PF-PF pelo app do Itaú nem do Nubank.

**Comment by mileo** [comment]
Pelo que eu sei esta intermitente.

https://tecnoblog.net/410981/pix-fora-do-ar-transferencia-gratis-nao-funciona-no-nubank-itau-e-outros-bancos/

**Comment by nekojess1** [comment]
Pelo visto sim, mas já voltou.

**Comment by andersondsl** [comment]
É incrível como em 2021, o BACEN desenvolveu literalmente uma rede de pagamentos centralizada.
Já temos 12 anos de blockchain rodando e entregando valor com um uptime de 100%.

**Comment by jhonatantft** [comment]
Bem colocado @andersondsl, no cenário atual vemos a descentralização de vários aspectos que afetam diretamente a sociedade, incluindo o sistema financeiro como mencionado.
Também estou na expectativa da maior adesão de blockchains de forma sistemática a fim de alcançar uma alta taxa da participação popular e democratização dos sistema em busca da liberdade financeira.

**Comment by PcXSk8** [comment]
Não estou conseguindo pagar PIX realizando a leitura com app do C6 bank.
sabem de algo?

**Comment by grudnitzki** [comment]
Pra mim também está fora do ar. Não confirma as transações

**Comment by renatofrota** [comment]
> Pra mim também está fora do ar. Não confirma as transações

O C6 especificamente?

**Comment by grudnitzki** [comment]
Testei transferir do Itaú para o Banco do Brasil. Deu erro ao confirmar

**Comment by PcXSk8** [comment]
> > Pra mim também está fora do ar. Não confirma as transações
> 
> O C6 especificamente?

Exato, consegui falar com o time lá e de fato havia uma instabilidde.

**Comment by PcXSk8** [comment]
Caros, o Pix está passando por alguma instabildade ?
Desde as 15:00 estou com alguns erros.
